### ISSUE TITLE
[Reports] Attachments break Report

### ISSUE THREAD

**Issue by Kowlin** [Cog-Creators/Red-DiscordBot#1816]
# Command bugs

#### Command name

``report``

#### What cog is this command from?

``Reports``

#### What were you expecting to happen?

T_T No answer needed...

#### What actually happened?

Following traceback occurred when bot failed to grab attachment from the message when message was deleted.

```
Exception in command 'report'
Traceback (most recent call last):
  File "/root/.local/lib/python3.6/site-packages/discord/ext/commands/core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/bot/redbot/cogs/reports/reports.py", line 254, in report
    val = await self.send_report(_m, guild)
  File "/bot/redbot/cogs/reports/reports.py", line 170, in send_report
    files = await Tunnel.files_from_attatch(msg)
  File "/bot/redbot/core/utils/tunnel.py", line 154, in files_from_attatch
    await a.save(_fp)
  File "/root/.local/lib/python3.6/site-packages/discord/message.py", line 103, in save
    data = yield from self._http.get_attachment(self.url)
  File "/root/.local/lib/python3.6/site-packages/discord/http.py", line 232, in get_attachment
    raise Forbidden(resp, 'cannot retrieve attachment')
discord.errors.Forbidden: Forbidden (status code: 403): cannot retrieve attachment

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/.local/lib/python3.6/site-packages/discord/ext/commands/bot.py", line 886, in invoke
    yield from ctx.command.invoke(ctx)
  File "/root/.local/lib/python3.6/site-packages/discord/ext/commands/core.py", line 932, in invoke
    yield from super().invoke(ctx)
  File "/root/.local/lib/python3.6/site-packages/discord/ext/commands/core.py", line 514, in invoke
    yield from injected(*ctx.args, **ctx.kwargs)
  File "/root/.local/lib/python3.6/site-packages/discord/ext/commands/core.py", line 71, in wrapped
    raise CommandInvokeError(e) from e
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: Forbidden: Forbidden (status code: 403): cannot retrieve attachment

```

#### How can we reproduce this issue?

use the ``report`` command while sending a attachment of any kind. Bonus points if you get locked out of creating any additional reports!

**Comment by mikeshardmind** [comment]
I'll have a fix for this pushed 1 moment

**Comment by tekulvw** [comment]
Fixed with #1822
